spec/services/shopify_import/creators/custom_collection_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+RSpec.describe ShopifyImport::Creators::CustomCollection, type: :service do
+  subject { described_class.new(shopify_data_feed) }
+
+  before { ShopifyAPI::Base.site = 'https://api_key:passowrd@shop_domain.myshopify.com/admin' }
+
+  describe '#save!', vcr: { cassette_name: 'shopify/base_custom_collection' } do
+    let(:shopify_custom_collection) { ShopifyAPI::CustomCollection.find(388_567_107) }
```

Comment:
I avoided making any network requests by having this variable directly built from factory in `shopify_import/creators/customer_spec.rb` (actually even without a factory, now looking at the code): https://github.com/spark-solutions/spree_shopify_importer/blob/43d7c3f0223552bd69f1cf01e094b50f2df69bd3/spec/services/shopify_import/creators/customer_spec.rb#L10